src/main/resources/static/js/subscription.js

```diff
@@ -511,18 +513,20 @@ jQuery(document).ready(function() {
                 success : function (data, textStatus) {
                     var returnData = [data];
                     if (returnData.length > 0) {
-                        $.jGrowl("Successful created subscription " + subscriptionJson.subscriptionName, {
+                        $.jGrowl("Subscriptions are successfully created", {
                             sticky : false,
                             theme : 'Error'
                         });
                         reload_table();
                     }
                 },
                 error : function (XMLHttpRequest, textStatus, errorThrown) {
-                    $.jGrowl("Failed to create Subscription: " + subscriptionJson.subscriptionName + " Error: " + XMLHttpRequest.responseText, {
-                        sticky : false,
-                        theme : 'Error'
-                    });
+                  reload_table();
+                  $.jGrowl("Failed to create next Subscriptions", {sticky: false, theme: 'Error'});
+                  var responseJSON = JSON.parse(XMLHttpRequest.responseText);
+                  for (var i = 0; i < responseJSON.length; i++) {
+                    $.jGrowl(responseJSON[i].subscription + " :: " + responseJSON[i].reason, {sticky: true, theme: 'Error'});
+                  }
```

Comment:
How will this work? If we have 30 subscriptions failure,, will the whole frontend  GUI be override of alot of Error popup windows. That will not fit a normal web browser window. We maybe should log all popup error windows that appers to these drop down menu lists(not used today) in top bar.

Reply:
Will be solved in a coming story.